src/screens/FormScreen.js

```diff
@@ -30,8 +45,13 @@ const FormScreen = ({ navigation }) => {
     setCost(cost)
   }
 
+  const addressInputHandler = (enteredAddress) => {
+    setEnteredAddress(enteredAddress)
+  }
+
   return (
       <View style={styles.formContainer}>
+        {Object.keys(userLocation).length === 0 ? renderManualInput() : null}
```

Comment:
@CHaiz15 This is the logic I was questioning.

Reply:
Looks good for now. We can look at refactoring later 👍